.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+### Jira Ticket: https://broadworkbench.atlassian.net/browse/[Ticket #]
+
+<!-- ### Dependencies --->
+<!-- Include any dependent tickets and describe the relationship. Include any relevant Jira tickets. --->
+
+## Summary of changes:
+<!--Please give an abridged version of the ticket description here and/or fill out the following fields.-->
+
+### What
+-
+
+### Why
+-
+
+### Testing strategy
+<!-- Note that changes impacting components in Storybook stories can be viewed at
+https://www.chromatic.com/library?appId=65fc89c9335768720ff8605a&branch=<branch>
+The branch corresponding to this PR is selected, and changes can be reviewed by commit. --->
```

Comment:
Storybook is only used for UI development (and specifically terra-ui, not sure what other UI repos have adopted it) so I'd remove this block.